### PR TITLE
PHP 8.4 support added

### PR DESCRIPTION
**Pull request by Atul-glo35265**
|    Q          |   A
|-------------- | ------
| Documentation | yes/no
| Bugfix        | yes/no
| BC Break      | yes/no
| New Feature   | yes/no
| RFC           | yes/no
| QA            | yes/no

### CODE REVIEW

**Comment by Atul-glo35265** [comment]
@gsteel, Please review this PR and do the needful.

**Comment by Atul-glo35265** [comment]
@froschdesign @gsteel Kindly review this PR and merge.

**Comment by Atul-glo35265** [comment]
@froschdesign @gsteel Kindly review and let us know, if any changes are required from our end.

**Comment by Atul-glo35265** [comment]
@froschdesign  @gsteel We understand things are busy, and reviews happen on your own time. Whenever you can, we'd appreciate a quick look to ensure we're on the right track. No worries if it's not immediate, just whenever it fits into your schedule!

File: composer.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
         }
     },
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0",
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```
Comment (froschdesign) [inline composer.json]: [Version 8.0 is no longer supported:](https://www.php.net/supported-versions)

```suggestion
        "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```

**Review by froschdesign — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (Atul-glo35265) [reply composer.json]: Done

**Review by gsteel — APPROVED** [review]
Thanks @Atul-glo35265 👍

**Pull request merged**